ext/cl_khr_semaphore.asciidoc

```diff
@@ -153,10 +153,9 @@ CL_SEMAPHORE_CONTEXT_KHR                                    0x2039
 CL_SEMAPHORE_REFERENCE_COUNT_KHR                            0x203A
 CL_SEMAPHORE_PROPERTIES_KHR                                 0x203B
 CL_SEMAPHORE_PAYLOAD_KHR                                    0x203C
+CL_SEMAPHORE_IS_EXPORTABLE_KHR                              0x2054
```

Comment:
I think it would be best to bump the extension version (to 0.9.2) since this is adding a new enum.